src/services/authentication/AvailabilityCheck.ts

```diff
@@ -21,8 +30,25 @@ export class AvailabilityCheck {
     return this.alive;
   }
 
-  private checkServerIsAlive(): void {
+  public checkServerIsAlive(): void {
     this.dmrService.readAttribute({address: [], name: "launch-type"})
-      .then(() => this.alive = true, () => this.alive = false);
+      .then((response) => {
+        if (!response) {
+          return undefined;
+        }
```

Comment:
Won't response always be a string? I'm not sure this check and return make sense